SmartSaver.MVC/Controllers/TransactionsController.cs

```diff
@@ -115,7 +115,7 @@ private Account GetAccountAuth()
         [HttpGet] 
         public ActionResult CreatePDF(TransactionViewModel transTime)
         {
-            var trans = GetAccountAuth().Transactions;
+            var trans = _context.Transactions.Where(a => a.AccountId.Equals(User.Identity.Name)).ToList();
```

Comment:
Produces empty list every single time here. User.Identiyty.Name holds UserId value, not AccountId